docker-compose/README.md

```diff
@@ -1,79 +1,118 @@
 # Docker Compose
```

Comment:
Do we have a section on making configuration changes like the parameters that @stephb9959  is interested in?

Reply:
Added a short hint about the configuration in general. I also added the default values being used in the env files so it is apparent which configuration options can be changed and so on.